File: orchalanguage-spring-integration-implementation/src/main/java/orcha/lang/configuration/Output.java
package orcha.lang.configuration;

public class Output {
    /**
     * MIME type like application/json.
     */
    String mimeType;
    /**
     * java.lang.Integer for instance. Primitive types like int forbidden.
     */
    String type;
    /**
     * Content of the output event often in Json format, should not be set programmatically.
     */
    String value;
    /**
     * Any Orcha adapter like {@link orcha.lang.configuration.JavaServiceAdapter}
     */
    ConfigurableProperties adapter;

    boolean autoStartup = true;
}

File: orchalanguage-spring-integration-implementation/src/main/kotlin/orcha/lang/configuration/Output.kt
package orcha.lang.configuration

class Output {
    /**
     * MIME type like application/json.
     */
    internal var mimeType: String? = null
    /**
     * java.lang.Integer for instance. Primitive types like int forbidden.
     */
    internal var type: String? = null
    /**
     * Content of the output event often in Json format, should not be set programmatically.
     */
    internal var value: String? = null
    /**
     * Any Orcha adapter like [orcha.lang.configuration.JavaServiceAdapter]
     */
    internal var adapter: ConfigurableProperties? = null

    internal var autoStartup = true
}



